composer.json

```diff
@@ -39,16 +39,16 @@
         }
     ],
     "require": {
-        "php": ">=7.0.13",
-        "illuminate/console": "5.2.* || 5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.*",
-        "illuminate/database": "5.2.* || 5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.*",
-        "illuminate/filesystem": "5.2.* || 5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.*"
+        "php": ">=7.1.3",
+        "illuminate/console": "^5.8",
+        "illuminate/database": "^5.8",
+        "illuminate/filesystem": "^5.8"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0",
+        "phpunit/phpunit": "^7.5",
         "mockery/mockery": "^1.0",
-        "orchestra/testbench": "^3.5",
-        "orchestra/database": "^3.5"
+        "orchestra/testbench": "^3.8",
+        "orchestra/database": "^3.8"
```

Comment:
`orchestra/testbench` `3.8` doesn't needs `orchestra/database`, you can simply remove the deps and service provider in your tests.

Reply:
@crynobone thanks for your advices, I removed what you said :)